.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Thanks for contributing to RIOT!
+-->
+
+
+### Preliminary checks
+
+<!--
+The RIOT community cares a lot about code quality.
+Therefore, before describing what your contribution is about, we would like you
+to answer the following questions:
+-->
+
+* Have you read the [contributing guide](../CONTRIBUTING.md) (and related wiki
```

Comment:
I'm not sure this link will resolve as you expect. Also: this link already appears on top of the text box for a new PR in a nice warning yellow. People who ignore that will most likely also ignore this line, so I think it is redundant.